PivovarAndrew/hometask-4/model/my_array_rotate.rb

```diff
@@ -0,0 +1,22 @@
+# !/usr/bin/env ruby
+# frozen_string_literal: true
+
+# Array class extension
+class Array
+  DEFAULT_ROTATE_QTY = 1
+
+  def lab_rotate(rotate_qty = nil)
+    rotated_array = dup
+    rotated_array.push(rotated_array.slice!(0, rotate_qty.nil? ? DEFAULT_ROTATE_QTY : calc_rotate_qty(rotate_qty)))
```

Comment:
I thinsk it will be array in array?

Reply:
there are also very convnient methotds
pop(2) - to fetch .ast 2 elements
and shift(2) - to fetch 2 first elements